content/blog/2017-11-16-ropensci-changes.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: "ropensci-changes"
+title: "changes: easy Git-based version control from R"
+date: 2017-11-16
+authors:
+  - name: Anikó B. Tóth (with edits from Nick Golding)
```

Comment:
add url that you'd like associated with you like this:

authors:
  \- name: Mark Padgham
      url: https://github.com/mpadge

I don't think there's a good way to include "(with edits from Nick Golding)" on authors line so you could either add him as second author, given that he also worked on the code for the package, or add an acknowledgement at the end, linking to his Twitter or GitHub.

For two authors:
authors:
  \- name: Anikó B. Tóth 
     url:
  \- name: Nick Golding
     url: https://twitter.com/_NickGolding_

Reply:
I changed this from two authors, because I thought it was confusing to have both our names, but Aniko's voice. I'm happy to be an anonymous co-author though, or two be a second author. Your call @anikobtoth!